TAGradingServer/account/admin-rubric.php

```diff
@@ -527,7 +527,7 @@ function calculatePercentageTotal() {
         var ec = 0;
         $('select.points').each(function(){
             var elem = $(this).attr('name').replace('point','ec');
-            if (!$('[name="'+elem+'"]').prop('checked') == true) {
+            if (!$('[name="'+elem+'"]').is(':checked') == true) {
```

Comment:
I am unclear about the reason for this change.  A quick skim of http://api.jquery.com/prop/ suggests that either statement would be fine.  Although this test (https://jsperf.com/prop-vs-ischecked/5) suggests that `.prop('checked')` processes faster (at least on Mac OS X browsers Safari, Chrome, and Firefox).